conda_environment.yml

```diff
@@ -4,6 +4,7 @@ channels:
 dependencies:
 - dask-core >2.0
 - dill
+- fsspec
```

Comment:
Is it necessary to specify fsspec?

Reply:
I don't think it is... but some of the tests were failing because it wasn't included.  I don't know enough about how the tests work but it seemed to help some of them.